modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -672,18 +1052,837 @@ export default class ortcRTCPeerConnection extends yaeti.EventTarget {
+                            id: info.rtxSsrc,
+                            attribute: 'cname',
+                            value: CNAME
+                        });
+
+                        mediaObject.ssrcs.push({
+                            id: info.rtxSsrc,
+                            attribute: 'msid',
+                            value: `${streamId} ${track.id}`
+                        });
+
+                        mediaObject.ssrcs.push({
+                            id: info.rtxSsrc,
+                            attribute: 'mslabel',
+                            value: streamId
+                        });
+
+                        mediaObject.ssrcs.push({
+                            id: info.rtxSsrc,
+                            attribute: 'label',
+                            value: track.id
+                        });
+
+                        mediaObject.ssrcGroups.push({
+                            semantics: 'FID',
+                            ssrcs: `${info.ssrc} ${info.rtxSsrc}`
+                        });
+                    }
+                }
+
+                // RTP header extensions.
+                mediaObject.ext = [];
+
+                for (const extension of localCapabilities.headerExtensions) {
+                    if (extension.kind && extension.kind !== kind) {
+                        continue; // eslint-disable-line no-continue
+                    }
+
+                    mediaObject.ext.push({
+                        value: extension.preferredId,
+                        uri: extension.uri
+                    });
+                }
+
+                // a=rtcp-mux attribute.
+                mediaObject.rtcpMux = 'rtcp-mux';
+
+                // a=rtcp-rsize.
+                mediaObject.rtcpRsize = 'rtcp-rsize';
+            }
+
+            // Add the media section.
+            sdpObject.media.push(mediaObject);
+        }
+    }
+
+    /**
+     * Start sending our media to the remote.
+     */
+    _sendMedia() {
+        logger.debug('_sendMedia()');
+
+        for (const info of this._localTrackInfos.values()) {
+            // Ignore if already sending.
+            if (info.sending) {
+                continue; // eslint-disable-line no-continue
+            }
+
+            // Update sending field.
+            info.sending = true;
+
+            const rtpSender = info.rtpSender;
+            const ssrc = info.ssrc;
+            const rtxSsrc = info.rtxSsrc;
+            const track = rtpSender.track;
+            const kind = track.kind;
+
+            const parameters = this._getParametersForRtpSender(kind, {
+                ssrc,
+                rtxSsrc
+            });
+
+            logger.debug(
+                'calling rtpSender.send(), parameters:', parameters);
+
+            // Start rsending media.
+            try {
+                rtpSender.send(parameters);
+            } catch (error) {
+                logger.error(`rtpSender.send() failed:${error}`);
```

Comment:
If so, it will try to send again for every new SDP O/A (which happens when a new participant enters or the room or leaves it). With same parameters so it will always throw the same error.